### DISCUSSION TITLE
Giving feature for query support

### DISCUSSION THREAD

**Discussion by nikhilnowfloats** [cachapa/firedart#119] [Q&A]
My app use firebase extensively. recently, our organisation is planning to release a windows version of the app as well. Is there any plan to give support to queries like limit, sort, where etc. ?

**Comment by cachapa** [comment]
I'm not working actively on Firedart, but I'd be happy to accept those features if someone wants to submit pull requests.

To be honest I don't think any of those features would be hard to implement since we're mostly wrapping Google's GRPC calls.

Maybe you can work on it? Be the change you want to see in the world, as they say 🙂
